kroxylicious-api/pom.xml

```diff
@@ -26,8 +26,8 @@
         <libs.dir>libs</libs.dir>
     </properties>
 
-    <name>Kroxylicious API</name>
-    <description>generic APIs used throughout Kroxylicious</description>
+    <name>Filter API</name>
```

Comment:
We should revert to the generic name.

Reply:
I changed it, but didn't revert it, because it was a pretty _unhelpful_ description.